std-bits/base/src/main/java/org/enso/base/enso_cloud/EnsoSecretReader.java

```diff
@@ -55,13 +55,6 @@ static String readSecret(String secretId) {
   }
 
   private static String readValueFromString(String json) {
-    return json.substring(1, json.length() - 1)
-        .replace("\\\"", "\"")
-        .replace("\\n", "\n")
-        .replace("\\t", "\t")
-        .replace("\\r", "\r")
-        .replace("\\b", "\b")
-        .replace("\\f", "\f")
-        .replace("\\\\", "\\");
+    return json.substring(1, json.length() - 1).translateEscapes();
```

Comment:
I think it may be good to include a TODO here that JSON can actually contain the Unicode escapes - if we ever run into this being an issue it would be a valuable hint.